### PR TITLE
Skip `drawEndLogo` if `doAni` is enabled

### PR DESCRIPTION
**Pull request by illusionyy**

### CODE REVIEW

File: source/zip_util.c [hunk 1]
```diff
@@ -121,13 +121,12 @@ int extract_zip_gh(const char* zip_file, const char* dest_path)
 		zip_entry_openbyindex(zip, i);
 		name = strchr(zip_entry_name(zip), '/');
 
-		if (!zip_entry_isdir(zip) && name && !(strncmp(name, "/json/", 6) && strncmp(name, "/shn/", 5)))
```
Comment (bucanero) [inline source/zip_util.c:124]: the `json/` and `shn/` check was added so when unzipping the .Zip updates from the Cheat_repository only the codes are extracted and all the other repository files are not.

if the update-patches zip has some other requirements, we can adjust the code while keeping this filter check

Comment (illusionyy) [reply source/zip_util.c:124]: Yea new behavior is now it extracts everything from the zip file. I'll see what can be done

Comment (illusionyy) [reply source/zip_util.c:124]: Reverted back to previous code with just `/mc4/` added

File: source/main.c [hunk 2]
```diff
@@ -1347,7 +1351,7 @@ s32 main(s32 argc, const char* argv[])
 		// Propagate the updated window to the screen
 		SDL_RenderPresent(renderer);
 	}
-
+	if (gcm_config.doAni)
 	drawEndLogo();
```
Comment (bucanero) [inline source/main.c]:
```suggestion
	if (gcm_config.doAni)
		drawEndLogo();
```

File: source/main.c [hunk 3]
```diff
@@ -1347,8 +1351,8 @@ s32 main(s32 argc, const char* argv[])
 		// Propagate the updated window to the screen
 		SDL_RenderPresent(renderer);
 	}
-
-	drawEndLogo();
+	if (gcm_config.doAni)
```
Comment (bucanero) [inline source/main.c:1351]:
```suggestion

	if (gcm_config.doAni)
```

File: source/main.c [hunk 4]
```diff
@@ -550,6 +550,10 @@ void SetMenu(int id)
 			break;
 
 		case MENU_USER_BACKUP: //User Backup Menu
+			if (!show_dialog(1, "Updating will overwrite any old Cheats\\Patch files\nDo you want to continue?"))
```
Comment (bucanero) [inline source/main.c]: based on #42 request, I think it's better if we just add another option under "Settings" about "overwrite files on update" and then let the user enable or disable that option.

Let's remove the check here and I'll work on a new PR to add the option and let the app behave based on that setting.

Comment (illusionyy) [reply source/main.c]: Sure 👍

**Review by bucanero — APPROVED** [review]

**Pull request merged**